.github/workflows/windows_build.yml

```diff
@@ -12,11 +12,13 @@ jobs:
       fail-fast: false
     runs-on: windows-2019
     name: windows-2019-MSVC16
+    env:
+      BOOST_ROOT: C:\local\boost_1_74_0
```

Comment:
any reasons not to use the latest version? currently 1.76 (released about 1 month ago)

Reply:
https://community.chocolatey.org/packages/boost-msvc-14.2